### PR TITLE
Add Segway and genomedata

### LINKED ISSUE #1953: Add recipes for Genomedata and Segway
Could you please add recipes for the Genomedata dense genomic data storage system and the Segway semi-automated genome annotation software?

http://genomedata.hoffmanlab.org/
http://segway.hoffmanlab.org/

Genomedata is a prerequisite for Segway.

Thank you!

### PR DESCRIPTION
**Pull request by dpryan79**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core: I should note that this adds a number of dependencies in addition to Segway and Genomedata. I'm not sure if there's a best practice regarding that. If so, let me know. This should close #1953

### CODE REVIEW

File: recipes/hgtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+
+  run:
+    - python
+    - setuptools
+    - setuptools_scm
```
Comment (bgruening) [inline recipes/hgtools/meta.yaml]: Wondering if this is really needed.

File: recipes/segway/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    - numpy
+    - forked-path
+    - hdf5
+    - hgtools
+    - colorbrewer
+    - drmaa >=0.4a3
+
+  run:
+    - python
+    - setuptools
```
Comment (bgruening) [inline recipes/segway/meta.yaml]: See above.

**Comment by bgruening** [comment]
Submitting multiple packages at once is totally ok :)

File: recipes/genomedata/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    - forked-path
+    - pytables >=3.0
+    - textinput
+    - gcc
+    - hdf5
+    - hgtools
+
+  run:
+    - python
+    - setuptools
```
Comment (dpryan79) [inline recipes/genomedata/meta.yaml]: Probably not needed, good catch. I'll go through and remove those throughout.

For C-based packages that use gcc, are we normally requiring libgcc at the runtime? I wouldn't normally do that, but I've seen some discussion surrounding it here so I added it in.

Comment (bgruening) [inline recipes/genomedata/meta.yaml]: This heavily depends on your program, I was unable to come up with a rule or recommendation when to include libgcc and when not ... it is not needed everytime, but hard to tell when it is needed.

**Comment by dpryan79** [comment]
@bgruening Setuptools has been removed as a runtime dependency. Anything else that you've noticed?

**Comment by bgruening** [comment]
This looks fine! Thanks a bunch!

**Pull request merged**